index.d.ts

```diff
@@ -8,13 +10,17 @@ declare namespace Cypress {
     composed?: Boolean;
   };
 
+  type CommandOptions = {
+    timeout?: Number;
+  };
+
   interface Chainable<Subject> {
     shadowClick(options?: EventOptions): Chainable<Subject>;
     shadowContains(content: string): Chainable<Subject>;
     shadowEq(index: number): Chainable<Subject>;
-    shadowFind(selector: string): Chainable<Subject>;
+    shadowFind(selector: string, options: CommandOptions): Chainable<Subject>;
```

Comment:
Can options be an optional type and have a default timeout set by these functions?